shared/common-adapters/floating-menu/menu-layout/index.native.tsx

```diff
@@ -77,8 +77,8 @@ const MenuRow = (props: MenuRowProps) => (
               <>
                 <Icon
                   color={props.danger ? Styles.globalColors.redDark : Styles.globalColors.black_60}
-                  fontSize={16}
                   style={props.iconStyle}
+                  sizeType="Default"
```

Comment:
Doesn't it default to this?

Reply:
It does, I just wanted to make it a point that it's on purpose that it is this size, in case we inspect the code later.